### ISSUE TITLE
Change title of 'Clear' button to 'Clear Changes'

### ISSUE THREAD

**Issue by KateAThornley** [CDH-Studio/I-Talent#368]
**Is your feature request related to a problem? Please describe.**
In order to assure users understand what they're clearing on the edit profile forms, the following solution is to increase app accessibility 

**Describe the solution you'd like**
The name on the clear buttons should be changed to "Clear Changes".

**Additional context**
Through further discussion with Peter Crane, possible modification to Clear Changes button could be an added popup to confirm deletion of recent changes,
